src/tools/flash_opts.c

```diff
@@ -137,8 +135,8 @@ int flash_get_opts(struct flash_opts* o, int ac, char** av) {
         av++;
     }
 
-    // command and (optional) device name
-    while(ac >= 1) { // looks like for stlinkv1 the device name and command may be swaped - check both positions in all cases
+    // commandd
```

Comment:
Is it possible to do something with this comment for improving its understandability?

Reply:
hmm, looks like merge issue somehow, original comment should have been kept

Reply:
To me it looks like, as if this comment is obsolete now anyway.